src/node.js

```diff
@@ -27,6 +27,10 @@ const AWS_S3_ENDPOINT = process.env.AWS_S3_ENDPOINT
 const AWS_S3_ADDRESSING_STYLE = process.env.AWS_S3_ADDRESSING_STYLE
 const AWS_S3_SIGNATURE_VERSION = process.env.AWS_S3_SIGNATURE_VERSION
 
+const PIN_SILENT = isBooleanStringSet(process.env.PIN_SILENT)
+const PIN_WHITELIST_DIDS = process.env.PIN_WHITELIST_DIDS ? process.env.PIN_WHITELIST_DIDS.split(',') : null
+const PIN_WHITELIST_SPACES = process.env.PIN_WHITELIST_SPACES ? process.env.PIN_WHITELIST_SPACES.split(',') : null
```

Comment:
We have to be clear in the documentation that there can't be spaces here. e.g. not correct: `space1, space2`